### ISSUE TITLE
Donne Gyroscope

### ISSUE THREAD

**Issue by Adopo** [jean-emmanuel/open-stage-control#154]
Je t'en ai parlé en vrai mais je le remets ici. Il pourrait vraiment être intéressant de rajouter les données Gyroscopique dans ton application !

**Comment by jean-emmanuel** [comment]
Tout à fait, je creuserai ça dès que possible. Je ne serais pas sur le coup pour une petite semaine mais n'hésite pas à spammer !

**Comment by Adopo** [comment]
Ca marche je construit une interface pour une session type LGML donc je risque de revenir souvent ici :-)

**Comment by jean-emmanuel** [comment]
Ressource pour développement à venir : https://www.alsacreations.com/tuto/lire/1501-api-device-orientation-motion-acceleration.html

**Comment by Adopo** [comment]
Trop bien !!!! Merci beaucoup. J'ai hâte de tester ça dans la prochaine mise à jour ! Ça ouvre plein de possibilités ! Encore merci !
